### ISSUE TITLE
Warning: Local Variable infinit loop

### ISSUE THREAD

**Issue by a0ki** [can1357/NoVmp#13]
![baixados](https://user-images.githubusercontent.com/20178679/93251567-e308c280-f783-11ea-965c-25c874e7c8a0.png)

**Comment by AmazingTurtle** [comment]
Yeah same for me. Is there anything needed from @can1357 to further investigate this or can I support you in any way?
